tests/test_mask.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import pytest
+import geopandas as gpd
+import numpy as np
+
+sys.path.append('../modules/')
+from mask import check_polygon, check_pre_proj, check_post_proj, check_coords
+
+world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
+can_index = world[world.name == "Canada"].index
+can_geom = world.loc[can_index, 'geometry']
+
+@pytest.mark.parametrize('p,passed', [
+    ({'string'}, False),
+    ({5}, False),
+    (can_geom, True),
+    (world, False),
+    (gpd.GeoSeries(), False)])
+def test_check_polygon(p, passed):
+    if passed:
+        assert check_polygon(p)
+    else:
+        with pytest.raises((TypeError, ValueError)):
+            check_polygon(p)
+
+@pytest.mark.parametrize('p,passed', [
+    ({'string'}, False),
+    ({5}, False),
+    (can_geom, True),
+    (world, False),
```

Comment:
It's not clear to me why this is supposed to fail. Can you explain?

Reply:
`world` is a polygon from natural earth that contains every country, where I only want to mask using Canada. It will have a different area than expected in `WGS84`.